src/components/CommentsList.jsx

```diff
@@ -1,5 +1,31 @@
+import { useState, useEffect } from "react";
+import { getAllComments } from "../utils.js";
+import CommentCard from './CommentCard.jsx'
+
 function CommentsList() {
-    return <h1>CommentsList</h1>
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+
+useEffect(() => {
+    getAllComments(1).then((result) => {
```

Comment:
we should dynamically pass the article id in here - this will always fetch the comments associated with article 1 only